src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -40,24 +40,6 @@ GcsActorScheduler::GcsActorScheduler(
 }
 
 void GcsActorScheduler::Schedule(std::shared_ptr<GcsActor> actor) {
```

Comment:
Can you add a check to make sure the actor isn't associated with any node id and work id?

Reply:
fixed